templates/taxi/manufacturer_confirm_delete.html

```diff
@@ -0,0 +1,19 @@
+{% extends "base.html" %}
+
+{% block content %}
+  <h1>Do you want to delete {{ manufacturer.name }}?</h1>
+  
+  {% if manufacturer.cars.exists %}
```

Comment:
The `exists` method should be called as a function. Change `manufacturer.cars.exists` to `manufacturer.cars.exists()` to correctly check for the existence of related cars.